data/wild_hands.py

```diff
@@ -26,8 +39,35 @@ def __getitem__(self, idx):
         Returns:
             tuple: (image, target) where target is the same image.
         """
-        pass
+        img = Image.open(self.img_dirs[idx]).convert("RGB")
+        img = self.transform(img)
+
+        target = Image.open(self.label_dirs[idx]).convert("RGB")  # not sure mn el convert tbh
+        target = self.transform(target)
+        return img, target
+
+    def get_img_dirs(self):
+        imgs = []
+        labels = []
+        for dir_path, sub_dirs, files in os.walk(self.dataset_path):
+            for file in files:
+                if 'ask' in file:
```

Comment:
In labels, you should check if the format is in ```.png```.

Reply:
why ?

Reply:
it's an easier way and will allow future support for the class if we added new data.